### PR TITLE
gh-2591 Fix cypress tests

### PR DESCRIPTION
**Pull request by sbishel**
#### Summary
Fixes localpermissions to return 'true' for team permission check.

Fixes other cypress test issues where template form was being used

#### Ticket Link
https://github.com/mattermost/focalboard/issues/2591

### CODE REVIEW

**Comment by sbishel** [comment]
woohoo - The cypress tests passed....

**Review by wiggin77 — APPROVED** [review]
LGTM :+1:

**Comment by sbishel** [comment]
Merging because cypress tests are passing. Test failures are in server tests.

File: server/services/permissions/localpermissions/localpermissions.go [hunk 1]
```diff
@@ -27,9 +27,7 @@ func New(store permissions.Store, logger *mlog.Logger) *Service {
 }
 
 func (s *Service) HasPermissionToTeam(userID, teamID string, permission *mmModel.Permission) bool {
-	if userID == "" || teamID == "" || permission == nil {
-		return false
-	}
+	// Locally there is no team, so return true
```
Comment (wiggin77) [inline server/services/permissions/localpermissions/localpermissions.go:30]: @sbishel this change breaks unit tests.  See https://community.mattermost.com/core/pl/qym96zhiopro3px6es7axg9f1w

**Pull request merged**